pkg/kube/structured/structured_test.go

```diff
@@ -480,6 +481,50 @@ func TestPersistentVolExists(t *testing.T) {
 	}
 }
 
+func TestPersistentVolClaimExists(t *testing.T) {
+	type args struct {
+		kubeClientset kubernetes.Interface
+		name          string
+		namespace     string
+		expectedPhase string
+	}
+	// expectedPhase: Available|Bound|Released|Failed|Pending
+	persistentvolumeClaimName := "persistentvolumeclaim1"
+	tests := []struct {
+		name    string
+		args    args
+		wantErr bool
+	}{
+		{
+			name: "pvc found",
+			args: args{
+				kubeClientset: fake.NewSimpleClientset(getResource(t, persistentVolumeClaimType, persistentvolumeClaimName)),
+				name:          persistentvolumeClaimName,
+				namespace:     "",
+				expectedPhase: "",
+			},
+			wantErr: false,
+		},
+		{
+			name: "pvc not found Test",
+			args: args{
+				kubeClientset: fake.NewSimpleClientset(getResource(t, persistentVolumeClaimType, "testabc")),
+				name:          persistentvolumeClaimName,
+				namespace:     "",
+				expectedPhase: "",
+			},
+			wantErr: true,
+		},
```

Comment:
Could you add an expectedPhase to the first case, as well as a test case where the expectedPhase is not the actual phase?

Reply:
done